packages/typespec-test/test/authoring/generated/typespec-ts/src/outputModels.ts

```diff
@@ -138,7 +138,6 @@ export type JobStatusOutput =
   | "cancelling"
   | "partiallyCompleted";
 /** A collection of SupportedLanguage resources. */
-export type PagedSupportedLanguageOutput = Paged<SupportedLanguageOutput>;
+export type SupportedLanguagesOutput = Paged<SupportedLanguageOutput>;
```

Comment:
this is because of typespec compiler change here https://typespec.io/docs/release-notes/release-2024-07-16#typespeccompiler
`[#3793](https://github.com/microsoft/typespec/pull/3793) Do not carry over @friendlyName with model is or op is`